chapters/inheritance.tex

```diff
@@ -551,6 +552,16 @@ \subsection{Modifiers for Array Elements}\label{modifiers-for-array-elements}
 \end{lstlisting}
 This implies \lstinline!b[k].c[i].a[j] = j!, \lstinline!b[k].c[i].d = i!, and \lstinline!b[k].p = k!.  For \lstinline!c.a! the additional (outer) \lstinline!each! has no effect,
 but it is necessary for \lstinline!c.d!.
+
+Subscripts on types behave the same as array variables.
```

Comment:
By _subscripts on types_, do you mean specifying array dimensions on the type rather than after the variable name?

Reply:
Yes. Updated accordingly.

Reply:
Nice! Resolving.